src/device/simulator.ts

```diff
@@ -132,6 +132,18 @@ const initialDataLog = (): DataLog => ({
   data: [],
 });
 
+export type SensorsLogged = Record<string, boolean>;
+
+const SENSORS_LOGGED = "sensorsLogged";
+
+const getSensorsLogged = (): SensorsLogged => {
+  const sensorsLogged = sessionStorage.getItem(SENSORS_LOGGED);
```

Comment:
I'm not sure this is worth the trouble of actual session storage.

Reply:
Removed the use of session storage -> [`2dc5220`](https://github.com/microbit-foundation/python-editor-v3/pull/1021/commits/2dc52203f09a15db813d0952316697974d8ce5d4).